cmake/Modules/winconf-msvc.cmake

```diff
@@ -4,7 +4,7 @@ if (STACKTRACE)
         add_compile_options(-Oy-)
     endif (NOT "${WINXXBITS}" STREQUAL "Win64")
     add_compile_options(-Zi)
-    link_libraries(dbghelp.lib /DEBUG)
+    link_libraries(dbghelp -DEBUG)
```

Comment:
I think `.lib` is necessary.

Reply:
It won't work for me. It seems like it just ignores this lib since I have unresolved externals due to this lib missing.
BTW, it has no `.lib` in other place.

Reply:
it fails for me without `.lib`
```

main.cpp.obj : error LNK2019: unresolved external symbol __imp_StackWalk64 referenced in function "class QString const __cdecl straceWin::getBacktrace(void)" (?getBacktrace@straceWin@@YA?BVQString@@XZ)
main.cpp.obj : error LNK2019: unresolved external symbol __imp_SymCleanup referenced in function "class QString const __cdecl straceWin::getBacktrace(void)" (?getBacktrace@straceWin@@YA?BVQString@@XZ)
main.cpp.obj : error LNK2019: unresolved external symbol __imp_SymGetModuleInfo64 referenced in function "int __cdecl straceWin::EnumModulesCB(char const *,unsigned __int64,void *)" (?EnumModulesCB@straceWin@@YAHPEBD_KPEAX@Z)
main.cpp.obj : error LNK2019: unresolved external symbol __imp_SymGetLineFromAddr64 referenced in function "class QString __cdecl straceWin::getSourcePathAndLineNumber(void *,unsigned __int64)" (?getSourcePathAndLineNumber@straceWin@@YA?AVQString@@PEAX_K@Z)
main.cpp.obj : error LNK2019: unresolved external symbol __imp_SymInitialize referenced in function "class QString const __cdecl straceWin::getBacktrace(void)" (?getBacktrace@straceWin@@YA?BVQString@@XZ)
main.cpp.obj : error LNK2019: unresolved external symbol __imp_SymSetContext referenced in function "class QString const __cdecl straceWin::getBacktrace(void)" (?getBacktrace@straceWin@@YA?BVQString@@XZ)
main.cpp.obj : error LNK2019: unresolved external symbol __imp_SymFromAddr referenced in function "class QString const __cdecl straceWin::getBacktrace(void)" (?getBacktrace@straceWin@@YA?BVQString@@XZ)
main.cpp.obj : error LNK2019: unresolved external symbol __imp_SymEnumSymbols referenced in function "class QString const __cdecl straceWin::getBacktrace(void)" (?getBacktrace@straceWin@@YA?BVQString@@XZ)
src\app\qbittorrent.exe : fatal error LNK1120: 8 unresolved externals
ninja: build stopped: subcommand failed.
```

Reply:
I think `.lib` is necessary since link_libraries directly translates to link flags if specified `item` inside is not an entity in cmake scope

Reply:
It unnecessary according to CMake docs:
>**A plain library name:** The generated link line will ask the linker to search for the library (e.g. foo becomes -lfoo or foo.lib).

Unfortunately I found weird behavior when I rechecked it. When I build it from scratch (i.e. in clean build directory) I always get unresolved externals regardless of having this extension. But when I change it after failed build and rebuild again it succeeds.
Now I have a solution and I'll publish it after I do some tests.

Reply:
@jagannatharjun, can you test #11033?

Reply:
Works! Tested with msvc and msys